src/main/java/appeng/core/AEConfig.java

```diff
@@ -195,6 +198,10 @@ private AEConfig( final File configFile )
 		this.colorApplicatorBattery = this.get( "battery", "colorApplicator", this.colorApplicatorBattery ).getInt( this.colorApplicatorBattery );
 		this.matterCannonBattery = this.get( "battery", "matterCannon", this.matterCannonBattery ).getInt( this.matterCannonBattery );
 
+		this.controllerMaxX = this.get( "controller", "maxX", this.controllerMaxX ).getInt( this.controllerMaxX );
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "controller" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)